pyseir/inference/model_fitter.py

```diff
@@ -2,19 +2,14 @@
 import iminuit
 
 # TODO use JAX for numpy XLA acceleration
-# from jax.config import config
-# config.update("jax_enable_x64", True) # enable float64 precision
-import numpy as np
```

Comment:
np is used in the code below. did you mean to remove it?

Reply:
That was a duplicate call. It's still there. This all needs to be reordered, but that's for another PR.